app/views/layouts/application.html.haml

```diff
@@ -9,6 +9,7 @@
     %link{:href => "https://fonts.googleapis.com", :rel => "preconnect"}/
     %link{:crossorigin => "", :href => "https://fonts.gstatic.com", :rel => "preconnect"}/
     %link{:href => "https://fonts.googleapis.com/css2?family=Bebas+Neue&family=Titillium+Web:ital,wght@0,200;0,300;0,400;0,600;0,700;0,900;1,200;1,300;1,400;1,600;1,700&display=swap", :rel => "stylesheet"}/
+    %script{src: "https://js.stripe.com/v3/", crossorigin: "anonymous"}
```

Comment:
I think crossorigin needs to be the hostname of the ticketing host, but we should read up on  CSRF protections.